sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/global-namespace/build.sbt

```diff
@@ -22,8 +22,6 @@ npmDevDependencies in Compile += "expose-loader" -> "0.7.1"
 // as expected by the scalajs-react facade
 webpackConfigFile := Some(baseDirectory.value / "webpack.config.js")
 
-// Needed to make scalajs-bundler actually use the custom config file in fastOptJS
-enableReloadWorkflow := false
```

Comment:
Why didn't the `static` test fail after my changes? Shouldn't it fail because it now uses https://github.com/scalacenter/scalajs-bundler/blob/master/sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/static/prod.webpack.config.js the UglifyJsPlugin  hopefully resulting in a different file size?